pkg/nodebootstrap/legacy/ubuntu.go

```diff
@@ -59,7 +60,16 @@ func (b UbuntuBootstrapper) UserData() (string, error) {
 }
 
 func makeUbuntuConfig(spec *api.ClusterConfig, ng *api.NodeGroup) ([]configFile, error) {
-	clientConfigData, err := makeClientConfigData(spec, kubeconfig.HeptioAuthenticatorAWS)
+	supportsAWSEKS, err := utils.IsMinVersion(api.Version1_20, spec.Metadata.Version)
```

Comment:
Can we please switch unconditionally to `AWSEKSAuthenticator`? The heptio version shipped in the image is old and it eg. does not work with metadata v2 only (see this bug: https://bugs.launchpad.net/cloud-images/+bug/1925487 ). That's also what the default `bootstrap.sh` script in all (all=all k8s versions) latest images do.

Reply:
And what about people using older AMIs? If we do, IMHO this should be highlighted as a breaking change maybe.

Reply:
I checked the oldest available ami (ami-0a5f7a061dc1fb866 , eu-central-1, from 2020-07-20) for [1.16](https://eu-central-1.console.aws.amazon.com/ec2/v2/home?region=eu-central-1#Images:visibility=public-images;name=ubuntu-eks/k8s_1.16/images/*;ownerAlias=099720109477;sort=desc:name) and that has already `awsci 1.18.69-1ubuntu0.18.04.1` which works ([minimum required version is 1.16.156](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)). All other amis for all other k8s versions are newer. So this should be safe.

Reply:
oh, well.